client/signup/steps/design-picker/index.jsx

```diff
@@ -0,0 +1,89 @@
+/**
+ * External dependencies
+ */
+import PropTypes from 'prop-types';
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import { localize } from 'i18n-calypso';
+import { identity } from 'lodash';
+
+/**
+ * Internal dependencies
+ */
+import { recordTracksEvent } from 'calypso/lib/analytics/tracks';
+import StepWrapper from 'calypso/signup/step-wrapper';
+import { submitSignupStep } from 'calypso/state/signup/progress/actions';
+import DesignGrid from 'calypso/landing/gutenboarding/onboarding-block/design-selector/design-grid';
+
+/**
+ * Style dependencies
+ */
+import './style.scss';
+
+class DesignPickerStep extends Component {
+	static propTypes = {
+		goToNextStep: PropTypes.func.isRequired,
+		signupDependencies: PropTypes.object.isRequired,
+		stepName: PropTypes.string.isRequired,
+		translate: PropTypes.func,
+	};
+
+	static defaultProps = {
+		useHeadstart: true,
+		translate: identity,
+	};
+
+	pickDesign = ( selectedDesign ) => {
+		recordTracksEvent( 'calypso_signup_select_design', {
+			theme: `pub/${ selectedDesign?.theme }`,
+			template: selectedDesign?.template,
+		} );
+
+		this.props.submitSignupStep(
+			{
+				stepName: this.props.stepName,
+				selectedDesign,
+			},
+			{
+				selectedDesign,
+			}
+		);
+
+		this.props.goToNextStep();
+	};
+
+	renderDesignPicker() {
+		return <DesignGrid locale="en" onSelect={ this.pickDesign } />;
+	}
+
+	headerText() {
+		const { translate } = this.props;
+
+		return translate( 'Choose a design' );
+	}
+	subHeaderText() {
+		const { translate } = this.props;
+
+		return translate( 'Pick your favorite homepage layout. You can customize or change it later', {
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **20** times:
`translate( 'Pick your favorite homepage layout. You can customize or change it later.' )` *ES Score: 8*


:information_source: This string is probably long enough that it doesn't need a **context**. Are you sure you don't want to use a translator comment instead?